frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/sdk.ts

```diff
@@ -0,0 +1,52 @@
+import { useDocsUrl } from "metabase/common/hooks";
+import { getPlan } from "metabase/common/utils/plan";
+import { useSelector } from "metabase/lib/redux";
+import {
+  getLearnUrl,
+  getSetting,
+  getUpgradeUrl,
+} from "metabase/selectors/settings";
+
+export const useEmbeddingSettingsLinks = () => {
+  const plan = useSelector(state =>
+    getPlan(getSetting(state, "token-features")),
+  );
+
+  const addUtmToLink = (url: string) =>
+    `${url}?${new URLSearchParams({
+      utm_source: "product",
+      utm_medium: "docs",
+      utm_campaign: "embedding-sdk",
+      utm_content: "embedding-sdk-admin",
+      source_plan: plan,
+    })}`;
+
+  const quickStartUrl = addUtmToLink("https://metaba.se/sdk-quick-start");
+
+  const { url: activationUrl } = useDocsUrl(
+    "paid-features/activating-the-enterprise-edition",
+  );
+
+  const switchMetabaseBinariesUrl = addUtmToLink(activationUrl);
```

Comment:
nit: I'd group the some of the vars together since having a space between all of them makes no visual queue which variables are for what.


```suggestion
  const { url: activationUrl } = useDocsUrl(
    "paid-features/activating-the-enterprise-edition",
  );
  const switchMetabaseBinariesUrl = addUtmToLink(activationUrl);
```

By grouping some of them together it shows more of a relationship between a set of variables that they're not all individual useable links, since some are just intermediate variables.